core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/report/edit/ReportEditViewModel.kt

```diff
@@ -33,6 +35,8 @@ data class ReportEditUiState(
     val seriesTitleError: String? = null,
```

Comment:
This is still an issue (as well as yAxisError): this is a singular field, when in fact, there are multiple series. This needs to be a map/list type where an error is linked to the uid of the series e.g. show the error only on the series that has an error.